### PR TITLE
Home page improvement

### PR DESCRIPTION
**Pull request by Kyle199910**
Added initial set of FAQ questions and Footer

### CODE REVIEW

File: app/templates/FAQ.html [hunk 1]
```diff
@@ -12,61 +12,76 @@
...
     <h1>Frequency Asked Questions(FAQ)</h1>
 
-        <h4>Learn more about BESST, and Battery Energy <h4>
+        <h4>Learn more about BESST, Powerwalls and Battery Energy <h4>
 
     <div class="accordionMenu">
         <!-- 1st menu  -->
         <input type="checkbox" name="trg1" id="acc1">
-        <label for="acc1">Quisquam Doloremeos</label>
+        <label for="acc1">What and how does a Powerwall work</label>
```
Comment (Rolo123y) [inline app/templates/FAQ.html]: can be split into 2 questions:
- what is a bess?
- how does a bess work?

File: app/templates/FAQ.html [hunk 2]
```diff
@@ -12,61 +12,76 @@
...
         <div class="content">
             <div class="inner">
-                Iorem ipsum
+                During times of peak electricity demand like when air conditioners ramp up on hot days – many utilities charge customers extra since they have to ramp up expensive additional electricity production to meet the demand. These charges are called “time-of-use” or “peak” prices since they are tied solely to when a customer uses the electricity.  
+Energy storage works in similar ways, but since energy storage lets customers decide when to buy and when to use the power, a customer is freed from premium charges, increasing cost savings.
+            </div>
+        </div>
+        <!-- 7th menu -->
+        <input type="checkbox" name="trg1" id="acc7">
+        <label for="acc7">Can a powerwall be installed into a home already set up with solar panels?</label>
```
Comment (Rolo123y) [inline app/templates/FAQ.html]: reword to -> can a bess such as a Tesla Powerwall be ...

File: app/__init__.py [hunk 3]
```diff
@@ -3,7 +3,9 @@
...
 app = Flask(__name__)
-app.config.from_object('config.ProdConfig')
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///monthlysavings.db' 
+app.config['SECRET_KEY'] = "SOMERANDOMSTRING"
+
 db = SQLAlchemy(app)
 
-from app import routes
\ No newline at end of file
+from app import routes
```
Comment (Rolo123y) [inline app/__init__.py]: no need to commit this file.

File: app/templates/base.html [hunk 4]
```diff
@@ -86,6 +85,9 @@
         integrity="sha384-ho+j7jyWK8fNQe+A12Hb8AhRq26LrZ/JpcUGGOn+Y7RsweNrtN/tE3MoK7ZeZDyx"
         crossorigin="anonymous"></script>
 
+    <footer>
+        &copy BESST| 2022 Management Engineering Capstone Project
+    </footer>
 </body>
```
Comment (Rolo123y) [inline app/templates/base.html:93]: styles on the base page can be put into base.css

**Review by Rolo123y — CHANGES_REQUESTED** [review]

**Comment by gaurav613** [comment]
should this be closed?

**Comment by Rolo123y** [comment]
this pr can be removed. These changes have been merged into main already.

**Pull request closed without merging**